### PR TITLE
feat(opentofu-switch): final draft for publishing

### PR DESCRIPTION
**Pull request by Gowiem**
## Info

* Adds the final draft for publishing. Already reviewed in GDocs, so this I only need a final formatting + triple check spelling + grammar before publishing. Thanks folks!

### CODE REVIEW

File: content/updates/opentofu-early-adopters.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+visible: true
+draft: false
+title: "Upgrading to OpenTofu: An Easy Journey"
```
Comment (Gowiem) [inline content/updates/opentofu-early-adopters.md:4]: Any thoughts on the name? Any better suggestions?

File: content/updates/opentofu-early-adopters.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+author: Matt Gowie
+slug: opentofu-early-adopters
+date: 2024-04-04
```
Comment (Gowiem) [inline content/updates/opentofu-early-adopters.md:7]:
```suggestion
date: 2024-04-08
```

File: static/img/updates/opentofu-early-adopters/logo.png
[the remark this answers is not in the document]
Comment (Gowiem) [reply static/img/updates/opentofu-early-adopters/logo.png:1]: Yeah, good call!

[the remark this answers is not in the document]
Comment (Gowiem) [reply static/img/updates/opentofu-early-adopters/logo.png:1]: Working with our designer to allow us to provide a list view and a blog post view. May be done by EOW or early next 👍

File: content/updates/opentofu-early-adopters.md (hunk 1 above)
Comment (Gowiem) [reply content/updates/opentofu-early-adopters.md:4]: I like this better for sure 👍 

```suggestion
title: "From Terraform to OpenTofu: Why and How"
```

File: content/updates/opentofu-early-adopters.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+3. Moving to OpenTofu gave us the ability to keep on top of evolving tools while still remaining true to our beliefs. For instance, we want to be better about testing our IaC logic. TerraTest has been the Terraform standard, but requires writing Golang, which is an obstacle for many of our clients. We were excited to test out `terraform test` but this new functionality is only available in BSL licensed versions of Terraform. Luckily, by choosing OpenTofu, we gain access to this and other future functionality.
```
Comment (gberenice) [inline content/updates/opentofu-early-adopters.md:25]:
```suggestion
3. Moving to OpenTofu gave us the ability to keep on top of evolving tools while still remaining true to our beliefs. For instance, we want to be better about testing our IaC logic. TerraTest has been the Terraform standard, but requires writing Golang, which is an obstacle for many of our clients. We were excited to test out `terraform test` but this new functionality is only available in BSL licensed versions of Terraform. Luckily, by choosing OpenTofu, we gain access to the [test command](https://opentofu.org/docs/cli/commands/test/) and other future functionality.
```
Let's provide a link to confirm that this is a released feature in OpenTofu.

File: content/updates/opentofu-early-adopters.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+Our migration journey has been very positive. We’ve only hit a couple minor issues that were quickly addressed:
+
+1. GPG Key signing support for non-mainstream providers is still a WIP. This results in tiny warnings like the screenshot; we find these to be an eyesore and prefer our logs to be clean to avoid missing any true issues. Luckily, for such providers, we’ve opened issues and it appears these will be fixed very soon:
+    1. [https://github.com/carlpett/terraform-provider-sops/issues/115](https://github.com/carlpett/terraform-provider-sops/issues/115) – Fixed!
+    2. [https://github.com/cloudposse/terraform-provider-utils/issues/344](https://github.com/cloudposse/terraform-provider-utils/issues/344) ([@kevcube](https://github.com/kevcube) is a Masterpoint engineer) – Fixed!
+    3. [https://github.com/tailscale/terraform-provider-tailscale/issues/335](https://github.com/tailscale/terraform-provider-tailscale/issues/335) – Open (as of the time of writing)
+
+![An example GPG Key Warning](/img/updates/opentofu-early-adopters/gpg-key-warning.png "An example GPG Key Warning")
+
```
Comment (gberenice) [inline content/updates/opentofu-early-adopters.md:95]: The list ordering is broken. I believe we need to remove these empty lines and add indentation before the image.

**Review by gberenice — COMMENTED** [review]
A couple of minor things + [the diagram issue](https://masterpoint.slack.com/archives/C04ATBLEJAV/p1712201996952569?thread_ts=1712160128.952779&cid=C04ATBLEJAV), and after resolving - let's :shipit:!

File: content/updates/opentofu-early-adopters.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+- The **[terraform-spacelift-cloud-infrastructure-automation](https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation) module**: We utilize this module to automate Spacelift, so we don’t need to ClickOps any changes in the Spacelift UI except for “Apply” confirmations. We write Atmos YAML stack files, which gives us a completely automated Spacelift platform.
+
+To give you an idea of the size of the project moved:
+
+- Our IaC manages 2386 resources across AWS, GitHub, Tailscale, Google Cloud, and many other providers.
+- We manage over 90 workspaces / state files.
+- This project has over 39K lines of Terraform code across over 480 files.
+
+To give you  a visual understanding, here is our internal TF Automation diagram for Spacelift:
```
Comment (gberenice) [inline content/updates/opentofu-early-adopters.md:47]:
```suggestion
To give you a visual understanding, here is our internal TF Automation diagram for Spacelift:
```

File: content/updates/opentofu-early-adopters.md (hunk 4 above)
Comment (Gowiem) [reply content/updates/opentofu-early-adopters.md:95]: Great catch!

File: content/updates/opentofu-early-adopters.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+![An example GPG Key Warning](/img/updates/opentofu-early-adopters/gpg-key-warning.png "An example GPG Key Warning")
+
+1. About 3 weeks into our usage of OpenTofu, we ran into an issue related using a recently published version of one of our own modules, [masterpointio/terraform-aws-tailscale](https://github.com/masterpointio/terraform-aws-tailscale). The problem was discussed[(you can view it in the OpenTofu Slack here)](https://opentofucommunity.slack.com/archives/C05R0FD0VRU/p1708463312128849) but the core of the issue is that a hiccup in the OpenTofu registry caused our newly published module to not show up until the registry was manually poked. The OpenTofu team was very responsive and helpful in getting this solved and it was fixed within a few hours. We were very appreciative of this short time to resolution. To me, it seems like that team is continuing to work through the outlier bugs in the registry which is understandable considering how new it is. Given the quick fix too, I find this to be a very acceptable problem!
```
Comment (Gowiem) [inline content/updates/opentofu-early-adopters.md:96]:
```suggestion
    3. [https://github.com/tailscale/terraform-provider-tailscale/issues/335](https://github.com/tailscale/terraform-provider-tailscale/issues/335) – Open (as of the time of writing)
      ![An example GPG Key Warning](/img/updates/opentofu-early-adopters/gpg-key-warning.png "An example GPG Key Warning")
1. About 3 weeks into our usage of OpenTofu, we ran into an issue related using a recently published version of one of our own modules, [masterpointio/terraform-aws-tailscale](https://github.com/masterpointio/terraform-aws-tailscale). The problem was discussed[(you can view it in the OpenTofu Slack here)](https://opentofucommunity.slack.com/archives/C05R0FD0VRU/p1708463312128849) but the core of the issue is that a hiccup in the OpenTofu registry caused our newly published module to not show up until the registry was manually poked. The OpenTofu team was very responsive and helpful in getting this solved and it was fixed within a few hours. We were very appreciative of this short time to resolution. To me, it seems like that team is continuing to work through the outlier bugs in the registry which is understandable considering how new it is. Given the quick fix too, I find this to be a very acceptable problem!
```

**Review by gberenice — APPROVED** [review]
:shipit: !

**Pull request merged**